schemas/v2.0.0.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+module.exports = {
+  version: '2.0.0',
+  baseUrl: 'https://prod-1-dc-api-oopeix3r.deepcrawl.com',
```

Comment:
`baseUrl` is back! Still don't think it belongs here since this schema is not **environment** specific. This url should always be passed from an env-specific config